imgproxy/templates/serviceaccount.yaml

```
@@ -0,0 +1,9 @@
+{{- if .Values.serviceAccount.enabled -}}
+{{- with .Values.serviceAccount -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .name | quote }}
+  annotations: {{ toYaml .annotations | nindent 4 }}
```

IMO asking a user to to [check the AWS docs](https://docs.aws.amazon.com/eks/latest/userguide/specify-service-account-role.html) and write annotation is error-prone.

We could do better here like

```suggestion
  annotations:
    {{ template "aws.iamRoleAnnotation" . }}
```

And then define the `aws.iamRole` by ourselves as a helper at `imgproxy/template/helpers.tpl`

```golang
{{/*
Template to build service account annotation for AWS IAM role
https://docs.aws.amazon.com/eks/latest/userguide/specify-service-account-role.html
*/}}
{{- define "aws.iamRoleAnnotation" }}
{{- printf "eks.amazonaws.com/role-arn: arn:aws:iam::%s:role/%s" .Values.awsAccountId .Values.awsIAMRoleName -}}
{{- end }}
```